lib/fauxpaas/command.rb

```diff
@@ -57,9 +59,14 @@ def action
 
     def execute
       signature = instance.signature(reference)
-      release = Release.new(signature: signature, fs: Fauxpaas.filesystem)
+
+      release = Release.new(
+        artifact: Artifact.new(signature: signature, ref_repo: Fauxpaas.ref_repo),
+        deploy_config: DeployConfig.from_ref(signature.deploy, Fauxpaas.ref_repo)
+      )
```

Comment:
Per some earlier discussion, I think it's totally fine for Release to lose responsibility for translating signature and fs into an artifact and deploy_config. That was the extent of what it was using those parameters for, and doing so with no Release-specific knowledge or conditionality. A separate factory to use here would be appropriate if we needed a seam between DeployCommand and Release, but this is quite clean and clear for now.